libs/pdi-scanner/src/rust/client.rs

```diff
@@ -42,35 +38,12 @@ pub struct ImageCalibrationTables {
     pub back_black: Vec<u8>,
 }
 
-macro_rules! recv {
```

Comment:
I removed the macros because they were less needed now that the timeout code is removed. It does make the methods a bit more verbose, but hopefully also more easily comprehensible.